src/uv-common.c

```diff
@@ -1002,3 +1002,10 @@ uint64_t uv_metrics_idle_time(uv_loop_t* loop) {
     idle_time += uv_hrtime() - entry_time;
   return idle_time;
 }
+
+void uv__msan_unpoison(const volatile void *a, size_t size) {
```

Comment:
I think this is better as a `static inline` in the header, so it actually generates no code if msan is disabled.